scripts.js

```diff
@@ -30,7 +30,6 @@ const loader = document.querySelector('.loader');
 //COMMON FUNCTIONS
 //fetch data from the URL
 async function fetchData(url) {
-
```

Comment:
no worry about this, I have a Pretier https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode installed to automatically format the code :D